env_setup.sh

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
What is this for?

Reply:
This file just specifies the necessary steps to build/deploy the kfserving charm. I guess it isn't really needed since this information can be aquired from integration.yaml . Should I remove this from the commit?

Reply:
Yeh, I think remove this from the repo

Reply:
+1 this is only going to cause confusion.

Reply:
done!